index.js

```diff
@@ -85,7 +86,10 @@ class ShareSheet extends React.Component {
   render(){
     return (
       <Overlay visible={this.props.visible} {...this.props}>
-        <View style={styles.actionSheetContainer}>
+        <View style={[styles.actionSheetContainer, 
+        {
+          height: this.props.visible ? Dimensions.get('window').height : 0
+        }]}>
```

Comment:
Instead of "removing" the height here, you can fix Overlay. Which seems to be a better fix to me. What do you think?
https://github.com/wasedaigo/react-native-share/commit/398aa738c8d3fc7de4f43f1c158dc3866313b1c2